config_sample.py

```diff
@@ -16,7 +16,7 @@
 GITHUB_DEPLOY_KEY = ''
 GITHUB_CI_KEY = ''
 GITHUB_CLIENT_ID = ''
-GITHUB_CLIENT_SECRET = ''
+GITHUB_CLIENT_KEY = ''
```

Comment:
Why did you change the name from **GITHUB_CLIENT_SECRET** to **GITHUB_CLIENT_KEY** ?

Reply:
`dodgy` doesn't allow having words such as "secret", it would give errors during GH actions runs.

Reply:
Ok.Got it.